packages/nimble-components/src/dialog/specs/README.md

```diff
@@ -157,7 +198,7 @@ By using the native `dialog` element, we get good a11y behavior without having t
 -   ESC key closes the dialog
 -   The native dialog defaults to the a11y role `dialog` (but we will override this, as discussed below)
 
-The role `alertdialog` should be used for most modal dialogs (since they demand the user's attention), so we will set it by default. When role is `alertdialog`, `aria-describedby` is also supposed to be set to reference the element containing the alert message (which would be part of the user-provided content). Unfortunately, because the `dialog` and the alert message element are on different sides of the shadow DOM boundary, we can't reference the element ID. Similarly, `aria-labelledby` is supposed to point to an element that is the title for the dialog, but it has the same problem with referencing IDs. Instead, `aria-label` should provide the title directly. It is up to the client to set `aria-label` appropriately, and we will synchronize it between `nimble-dialog` and the `dialog` element in the shadow root.
+The role `alertdialog` should be used for most modal dialogs (since they demand the user's attention), so we will set it by default. When role is `alertdialog`, `aria-describedby` should be set to reference the element containing the alert message. Therefore, `aria-describedby` will reference the `content` provided by the dialog. Similarly, `aria-labelledby` should point to an element that is the title for the dialog, so it will reference the `title` slot of the dialog. Therefore, a client should always provide an appropriate title for their dialog, even if `header-hidden` is `true`.
```

Comment:
nit: Can you emphasize the difference between what happen internally automatically by the nimble-dialog and what the user is responsible for? I was initially confused thinking it was describing that the client had to provide the aria-* attributes.

Reply:
I rewrote this section.